appendix/configure-env-vars.rst

```diff
@@ -38,17 +38,19 @@ Use these environment variables to configure Zammad's behavior at runtime.
 General Options
 ---------------
 
-APP_RESTART_CMD
-   The command Zammad will use to automatically restart the server
-   after :admin-docs:`changes have been made in the Object Manager </system/objects.html>`.
-   (*E.g.,* ``"systemctl restart zammad"``)
+AUTO_SHUTDOWN
```

Comment:
There's a misunderstanding here @ralf401. This is a Zammad setting, not an env var, so it must be set via the Rails console, if required.

Reply:
Oh thanks for the hint! I moved it now to the console section. Hope this is ok now.